src/components/SearchModal/CurrencySearchModal.tsx

```diff
@@ -1,4 +1,4 @@
-import { Currency } from '@uniswap/sdk'
+import { Currency } from 'uniswap-fuse-sdk'
```

Comment:
@mul53 can you give link to uniswap-fuse-sdk

Reply:
here is the [code](https://github.com/mul53/uniswap-sdk)

Reply:
can you change this once again to `@fuseio/fuse-swap-sdk`  :) ?

Reply:
all references ofc